Specify support (#460)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

:::

:::details
### Does Wasabi support Trezor's, Ledger's and Coldcard's hidden wallets?

> Partially. Only device side passphrase is supported. PC side passphrase is not. This means the hidden wallet feature can be used with Trezor T, Ledger Nano S and ColdCard, but it cannot be used with Trezor T. After the 12 or 24 words, enter the passphrase as the 13th or 25th word. It’s part of [BIP 39](/using-wasabi/BIPs.md#bip-39-mnemonic-code-for-generating-deterministic-keys).

Reviewer: > , but it cannot be used with Trezor T

Probably a typo, should be `Trezor One`

Author: Yes, thanks!

:::

:::details

> ### Does Wasabi support Trezor's, Ledger's and Coldcard's hidden wallets?

Reviewer 2: ```suggestion
### Does Wasabi support the hidden wallets of hardware wallets?
```

to keep it more maintaineable...
